layouts/partials/github_release_table_json.html

```diff
@@ -0,0 +1,29 @@
+{{ $repo := . }}
+{{ $prefix := "https://api.github.com/repos/kicad/" }}
+
+{{ $json := getJSON $prefix $repo "/releases" }}
```

Comment:
Hmm, the problem here would be we would need to periodically regenerate the website as getJSON is static one-time.

Reply:
Yep JavaScript is a much better option. The version above is no longer displayed and I will remove the code once a better version is implemented.